libraries/ESP8266WiFiMesh/src/JsonTranslator.cpp

```diff
@@ -0,0 +1,304 @@
+/*
```

Comment:
I would be happy to use an established lib, but since the common ones are quite large I thought that was discouraged and so just created what I needed. Should I import an established JSON lib instead?

Reply:
I'd +1 importing ArduinoJson as a submodule

Reply:
How should we do with this? Should we use ArduinoJson instead?

Reply:
@devyte bump

Reply:
Existing JSON serializer generalized.